style.css

```diff
@@ -30,6 +30,10 @@ h1{
 	width: 98%;
 }
 
+.both-boards {
```

Comment:
You can select an element by id like #both_boards (# instead of .), so you don't need to create a class that will only ever apply to a single element.